{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":13376,"level":50,"msg":"Error response from auth { error: 'expired token' }","time":"2021-01-14T11:28:10.955Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":13376,"level":50,"msg":"Error accesssing editSubcategory { error: \"Cannot read property 'id' of null\" }","time":"2021-01-14T11:31:18.711Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":13376,"level":50,"msg":"Error accesssing editSubcategory { error: \"Cannot read property 'dataValues' of undefined\" }","time":"2021-01-14T11:31:41.849Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":456,"level":50,"msg":"Error accesssing editSubcategory { error: \"Cannot read property 'dataValues' of undefined\" }","time":"2021-01-14T11:32:59.882Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":456,"level":50,"msg":"Error accesssing editSubcategory { error: \"Cannot read property 'id' of null\" }","time":"2021-01-14T11:35:01.840Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":596,"level":50,"msg":"Error accesssing createProduct {\n  error: TypeError: Cannot read property 'id' of null\n      at D:\\work\\user_dashboard\\routes\\productRoutes.js:28:74\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n}","time":"2021-01-14T11:42:44.019Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":596,"level":50,"msg":"Error accesssing createProduct {\n  error: TypeError: Cannot read property 'id' of null\n      at D:\\work\\user_dashboard\\routes\\productRoutes.js:28:74\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n}","time":"2021-01-14T11:43:39.676Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":16068,"level":50,"msg":"Error accesssing createProduct {\n  error: TypeError: Cannot read property 'id' of null\n      at D:\\work\\user_dashboard\\routes\\productRoutes.js:28:74\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n}","time":"2021-01-14T11:44:39.514Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":16068,"level":50,"msg":"Error accesssing createProduct {\n  error: TypeError: Cannot read property 'id' of null\n      at D:\\work\\user_dashboard\\routes\\productRoutes.js:28:74\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n}","time":"2021-01-14T11:47:39.014Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":16068,"level":50,"msg":"Error accesssing createProduct {\n  error: TypeError: Cannot read property 'id' of null\n      at D:\\work\\user_dashboard\\routes\\productRoutes.js:28:74\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n}","time":"2021-01-14T11:47:51.004Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":16068,"level":50,"msg":"Error accesssing createProduct {\n  error: TypeError: Cannot read property 'id' of null\n      at D:\\work\\user_dashboard\\routes\\productRoutes.js:28:74\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n}","time":"2021-01-14T11:49:34.953Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":13956,"level":50,"msg":"Error accesssing createProduct {\n  error: TypeError: Cannot read property 'id' of null\n      at D:\\work\\user_dashboard\\routes\\productRoutes.js:29:74\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n}","time":"2021-01-14T11:51:25.259Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":9920,"level":50,"msg":"Error accesssing createProduct {\n  error: ValidationError [SequelizeValidationError]: notNull Violation: Products.name cannot be null,\n  notNull Violation: Products.price cannot be null,\n  notNull Violation: Products.quantity cannot be null\n      at InstanceValidator._validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:78:13)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async InstanceValidator._validateAndRunHooks (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:111:7)\n      at async InstanceValidator.validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:93:12)\n      at async model.save (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:3877:7)\n      at async Function.create (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:2207:12)\n      at async D:\\work\\user_dashboard\\routes\\productRoutes.js:29:28 {\n    errors: [\n      [ValidationErrorItem],\n      [ValidationErrorItem],\n      [ValidationErrorItem]\n    ]\n  }\n}","time":"2021-01-14T11:53:01.190Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":9920,"level":50,"msg":"Error accesssing createProduct {\n  error: ValidationError [SequelizeValidationError]: notNull Violation: Products.name cannot be null,\n  notNull Violation: Products.price cannot be null,\n  notNull Violation: Products.quantity cannot be null\n      at InstanceValidator._validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:78:13)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async InstanceValidator._validateAndRunHooks (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:111:7)\n      at async InstanceValidator.validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:93:12)\n      at async model.save (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:3877:7)\n      at async Function.create (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:2207:12)\n      at async D:\\work\\user_dashboard\\routes\\productRoutes.js:29:28 {\n    errors: [\n      [ValidationErrorItem],\n      [ValidationErrorItem],\n      [ValidationErrorItem]\n    ]\n  }\n}","time":"2021-01-14T11:53:15.816Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":9920,"level":50,"msg":"Error accesssing createProduct {\n  error: ValidationError [SequelizeValidationError]: notNull Violation: Products.name cannot be null,\n  notNull Violation: Products.price cannot be null,\n  notNull Violation: Products.quantity cannot be null\n      at InstanceValidator._validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:78:13)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async InstanceValidator._validateAndRunHooks (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:111:7)\n      at async InstanceValidator.validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:93:12)\n      at async model.save (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:3877:7)\n      at async Function.create (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:2207:12)\n      at async D:\\work\\user_dashboard\\routes\\productRoutes.js:29:28 {\n    errors: [\n      [ValidationErrorItem],\n      [ValidationErrorItem],\n      [ValidationErrorItem]\n    ]\n  }\n}","time":"2021-01-14T12:00:30.345Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":9920,"level":50,"msg":"Error accesssing createProduct {\n  error: ValidationError [SequelizeValidationError]: notNull Violation: Products.name cannot be null,\n  notNull Violation: Products.price cannot be null,\n  notNull Violation: Products.quantity cannot be null\n      at InstanceValidator._validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:78:13)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async InstanceValidator._validateAndRunHooks (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:111:7)\n      at async InstanceValidator.validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:93:12)\n      at async model.save (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:3877:7)\n      at async Function.create (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:2207:12)\n      at async D:\\work\\user_dashboard\\routes\\productRoutes.js:29:28 {\n    errors: [\n      [ValidationErrorItem],\n      [ValidationErrorItem],\n      [ValidationErrorItem]\n    ]\n  }\n}","time":"2021-01-14T12:00:38.756Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":9920,"level":50,"msg":"Error accesssing createProduct {\n  error: ValidationError [SequelizeValidationError]: notNull Violation: Products.name cannot be null,\n  notNull Violation: Products.price cannot be null,\n  notNull Violation: Products.quantity cannot be null\n      at InstanceValidator._validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:78:13)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async InstanceValidator._validateAndRunHooks (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:111:7)\n      at async InstanceValidator.validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:93:12)\n      at async model.save (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:3877:7)\n      at async Function.create (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:2207:12)\n      at async D:\\work\\user_dashboard\\routes\\productRoutes.js:29:28 {\n    errors: [\n      [ValidationErrorItem],\n      [ValidationErrorItem],\n      [ValidationErrorItem]\n    ]\n  }\n}","time":"2021-01-14T12:00:53.715Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":9920,"level":50,"msg":"Error accesssing createProduct {\n  error: ValidationError [SequelizeValidationError]: notNull Violation: Products.name cannot be null,\n  notNull Violation: Products.price cannot be null,\n  notNull Violation: Products.quantity cannot be null\n      at InstanceValidator._validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:78:13)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async InstanceValidator._validateAndRunHooks (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:111:7)\n      at async InstanceValidator.validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:93:12)\n      at async model.save (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:3877:7)\n      at async Function.create (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:2207:12)\n      at async D:\\work\\user_dashboard\\routes\\productRoutes.js:29:28 {\n    errors: [\n      [ValidationErrorItem],\n      [ValidationErrorItem],\n      [ValidationErrorItem]\n    ]\n  }\n}","time":"2021-01-14T12:01:06.019Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":1556,"level":50,"msg":"Error accesssing createProduct {\n  error: ValidationError [SequelizeValidationError]: notNull Violation: Products.name cannot be null,\n  notNull Violation: Products.price cannot be null,\n  notNull Violation: Products.quantity cannot be null\n      at InstanceValidator._validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:78:13)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async InstanceValidator._validateAndRunHooks (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:111:7)\n      at async InstanceValidator.validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:93:12)\n      at async model.save (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:3877:7)\n      at async Function.create (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:2207:12)\n      at async D:\\work\\user_dashboard\\routes\\productRoutes.js:29:28 {\n    errors: [\n      [ValidationErrorItem],\n      [ValidationErrorItem],\n      [ValidationErrorItem]\n    ]\n  }\n}","time":"2021-01-14T12:01:45.482Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":15968,"level":50,"msg":"Error accesssing createProduct {\n  error: ValidationError [SequelizeValidationError]: Validation error: Please enter a value greater than zero,\n  Validation error: Validation notIn on status failed\n      at InstanceValidator._validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:78:13)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async InstanceValidator._validateAndRunHooks (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:111:7)\n      at async InstanceValidator.validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:93:12)\n      at async model.save (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:3877:7)\n      at async Function.create (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:2207:12)\n      at async D:\\work\\user_dashboard\\routes\\productRoutes.js:29:28 {\n    errors: [ [ValidationErrorItem], [ValidationErrorItem] ]\n  }\n}","time":"2021-01-14T12:02:29.076Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":15968,"level":50,"msg":"Error accesssing createProduct {\n  error: ValidationError [SequelizeValidationError]: Validation error: Please enter a value greater than zero,\n  Validation error: Validation notIn on status failed\n      at InstanceValidator._validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:78:13)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async InstanceValidator._validateAndRunHooks (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:111:7)\n      at async InstanceValidator.validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:93:12)\n      at async model.save (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:3877:7)\n      at async Function.create (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:2207:12)\n      at async D:\\work\\user_dashboard\\routes\\productRoutes.js:29:28 {\n    errors: [ [ValidationErrorItem], [ValidationErrorItem] ]\n  }\n}","time":"2021-01-14T12:02:49.958Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":15968,"level":50,"msg":"Error accesssing createProduct {\n  error: ValidationError [SequelizeValidationError]: Validation error: Please enter a value greater than zero,\n  Validation error: Validation notIn on status failed\n      at InstanceValidator._validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:78:13)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async InstanceValidator._validateAndRunHooks (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:111:7)\n      at async InstanceValidator.validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:93:12)\n      at async model.save (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:3877:7)\n      at async Function.create (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:2207:12)\n      at async D:\\work\\user_dashboard\\routes\\productRoutes.js:29:28 {\n    errors: [ [ValidationErrorItem], [ValidationErrorItem] ]\n  }\n}","time":"2021-01-14T12:02:58.951Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":15968,"level":50,"msg":"Error accesssing createProduct {\n  error: ValidationError [SequelizeValidationError]: Validation error: Please enter a value greater than zero,\n  Validation error: Validation notIn on status failed\n      at InstanceValidator._validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:78:13)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async InstanceValidator._validateAndRunHooks (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:111:7)\n      at async InstanceValidator.validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:93:12)\n      at async model.save (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:3877:7)\n      at async Function.create (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:2207:12)\n      at async D:\\work\\user_dashboard\\routes\\productRoutes.js:29:28 {\n    errors: [ [ValidationErrorItem], [ValidationErrorItem] ]\n  }\n}","time":"2021-01-14T12:03:10.934Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":15968,"level":50,"msg":"Error accesssing createProduct {\n  error: ValidationError [SequelizeValidationError]: Validation error: Validation notIn on status failed\n      at InstanceValidator._validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:78:13)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async InstanceValidator._validateAndRunHooks (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:111:7)\n      at async InstanceValidator.validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:93:12)\n      at async model.save (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:3877:7)\n      at async Function.create (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:2207:12)\n      at async D:\\work\\user_dashboard\\routes\\productRoutes.js:29:28 {\n    errors: [ [ValidationErrorItem] ]\n  }\n}","time":"2021-01-14T12:03:20.368Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":10092,"level":50,"msg":"Error accesssing editProduct {\n  error: ValidationError [SequelizeValidationError]: Validation error: Validation isIn on status failed\n      at InstanceValidator._validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:78:13)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async InstanceValidator._validateAndRunHooks (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:111:7)\n      at async InstanceValidator.validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:93:12)\n      at async Function.update (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:3076:26)\n      at async D:\\work\\user_dashboard\\routes\\productRoutes.js:65:35 {\n    errors: [ [ValidationErrorItem] ]\n  }\n}","time":"2021-01-14T12:10:22.847Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":10092,"level":50,"msg":"Error accesssing editProduct {\n  error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n      at ServerResponse.setHeader (_http_outgoing.js:558:11)\n      at ServerResponse.header (D:\\work\\user_dashboard\\node_modules\\express\\lib\\response.js:771:10)\n      at ServerResponse.send (D:\\work\\user_dashboard\\node_modules\\express\\lib\\response.js:170:12)\n      at ServerResponse.json (D:\\work\\user_dashboard\\node_modules\\express\\lib\\response.js:267:15)\n      at ServerResponse.send (D:\\work\\user_dashboard\\node_modules\\express\\lib\\response.js:158:21)\n      at D:\\work\\user_dashboard\\routes\\productRoutes.js:87:25\n      at processTicksAndRejections (internal/process/task_queues.js:93:5) {\n    code: 'ERR_HTTP_HEADERS_SENT'\n  }\n}","time":"2021-01-14T12:10:34.720Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":16060,"level":50,"msg":"Error accessing getList { error: 'Subcategories is not associated to Categories!' }","time":"2021-01-14T12:24:30.889Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":13680,"level":50,"msg":"Error accessing getList { error: 'Categories is not associated to Subcategories!' }","time":"2021-01-14T12:25:31.406Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":17332,"level":50,"msg":"Error accessing getList { error: \"Unknown column 'Subcategories.CategoryId' in 'field list'\" }","time":"2021-01-14T12:27:25.281Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":6556,"level":50,"msg":"Error accessing getList { error: \"Unknown column 'Subcategories.CategoryId' in 'field list'\" }","time":"2021-01-14T12:32:29.589Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":6556,"level":50,"msg":"Error accessing getList { error: \"Unknown column 'Subcategories.CategoryId' in 'field list'\" }","time":"2021-01-14T12:32:30.938Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":7220,"level":50,"msg":"Error accesssing createProduct {\n  error: UniqueConstraintError [SequelizeUniqueConstraintError]: Validation error\n      at Query.formatError (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:218:16)\n      at Query.run (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:54:18)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\sequelize.js:619:16\n      at async MySQLQueryInterface.insert (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:734:21)\n      at async model.save (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:3954:35)\n      at async Function.create (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:2207:12)\n      at async D:\\work\\user_dashboard\\routes\\productRoutes.js:31:28 {\n    errors: [ [ValidationErrorItem] ],\n    fields: { 'products.name': 'nike casual shoes' },\n    parent: Error: Duplicate entry 'nike casual shoes' for key 'products.name'\n        at Packet.asError (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n        at Execute.execute (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n        at Connection.handlePacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:425:32)\n        at PacketParser.onPacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:75:12)\n        at PacketParser.executeStart (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n        at Socket.<anonymous> (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:82:25)\n        at Socket.emit (events.js:315:20)\n        at addChunk (internal/streams/readable.js:309:12)\n        at readableAddChunk (internal/streams/readable.js:284:9)\n        at Socket.Readable.push (internal/streams/readable.js:223:10) {\n      code: 'ER_DUP_ENTRY',\n      errno: 1062,\n      sqlState: '23000',\n      sqlMessage: \"Duplicate entry 'nike casual shoes' for key 'products.name'\",\n      sql: 'INSERT INTO `Products` (`id`,`name`,`description`,`price`,`status`,`quantity`,`category_id`,`subcategory_id`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?,?,?);',\n      parameters: [Array]\n    },\n    original: Error: Duplicate entry 'nike casual shoes' for key 'products.name'\n        at Packet.asError (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n        at Execute.execute (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n        at Connection.handlePacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:425:32)\n        at PacketParser.onPacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:75:12)\n        at PacketParser.executeStart (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n        at Socket.<anonymous> (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:82:25)\n        at Socket.emit (events.js:315:20)\n        at addChunk (internal/streams/readable.js:309:12)\n        at readableAddChunk (internal/streams/readable.js:284:9)\n        at Socket.Readable.push (internal/streams/readable.js:223:10) {\n      code: 'ER_DUP_ENTRY',\n      errno: 1062,\n      sqlState: '23000',\n      sqlMessage: \"Duplicate entry 'nike casual shoes' for key 'products.name'\",\n      sql: 'INSERT INTO `Products` (`id`,`name`,`description`,`price`,`status`,`quantity`,`category_id`,`subcategory_id`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?,?,?);',\n      parameters: [Array]\n    },\n    sql: 'INSERT INTO `Products` (`id`,`name`,`description`,`price`,`status`,`quantity`,`category_id`,`subcategory_id`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?,?,?);'\n  }\n}","time":"2021-01-14T12:48:05.100Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":7220,"level":50,"msg":"Error response from auth { error: 'expired token' }","time":"2021-01-14T12:48:56.203Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":6060,"level":50,"msg":"Error accesssing getProducts {\n  error: ReferenceError: Products is not defined\n      at D:\\work\\user_dashboard\\routes\\productRoutes.js:137:33\n      at Layer.handle [as handle_request] (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\layer.js:95:5)\n      at next (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\route.js:137:13)\n      at getCategoryId (D:\\work\\user_dashboard\\middleware\\getCategoryId.js:25:9)\n      at Layer.handle [as handle_request] (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\layer.js:95:5)\n      at next (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\route.js:137:13)\n      at D:\\work\\user_dashboard\\middleware\\admin.js:23:17\n      at Layer.handle [as handle_request] (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\layer.js:95:5)\n      at next (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\route.js:137:13)\n      at authenticateToken (D:\\work\\user_dashboard\\middleware\\auth.js:47:17)\n}","time":"2021-01-14T13:48:24.442Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":16520,"level":50,"msg":"Error accesssing getProducts {\n  error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n      at ServerResponse.setHeader (_http_outgoing.js:558:11)\n      at ServerResponse.header (D:\\work\\user_dashboard\\node_modules\\express\\lib\\response.js:771:10)\n      at ServerResponse.send (D:\\work\\user_dashboard\\node_modules\\express\\lib\\response.js:170:12)\n      at ServerResponse.json (D:\\work\\user_dashboard\\node_modules\\express\\lib\\response.js:267:15)\n      at ServerResponse.send (D:\\work\\user_dashboard\\node_modules\\express\\lib\\response.js:158:21)\n      at D:\\work\\user_dashboard\\routes\\productRoutes.js:139:29\n      at processTicksAndRejections (internal/process/task_queues.js:93:5) {\n    code: 'ERR_HTTP_HEADERS_SENT'\n  }\n}","time":"2021-01-14T13:50:53.786Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":12216,"level":50,"msg":"Error accesssing getProducts {\n  error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n      at ServerResponse.setHeader (_http_outgoing.js:558:11)\n      at ServerResponse.header (D:\\work\\user_dashboard\\node_modules\\express\\lib\\response.js:771:10)\n      at ServerResponse.send (D:\\work\\user_dashboard\\node_modules\\express\\lib\\response.js:170:12)\n      at ServerResponse.json (D:\\work\\user_dashboard\\node_modules\\express\\lib\\response.js:267:15)\n      at ServerResponse.send (D:\\work\\user_dashboard\\node_modules\\express\\lib\\response.js:158:21)\n      at D:\\work\\user_dashboard\\routes\\productRoutes.js:139:29\n      at processTicksAndRejections (internal/process/task_queues.js:93:5) {\n    code: 'ERR_HTTP_HEADERS_SENT'\n  }\n}","time":"2021-01-14T13:54:26.529Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":12216,"level":50,"msg":"Error response from getCategoryId { error: 'unauthorized role' }","time":"2021-01-14T13:55:20.956Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":8828,"level":50,"msg":"Error accesssing getProducts {\n  error: DatabaseError [SequelizeDatabaseError]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''1'' at line 1\n      at Query.formatError (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:239:16)\n      at Query.run (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:54:18)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\sequelize.js:619:16\n      at async MySQLQueryInterface.select (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:938:12)\n      at async Function.findAll (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:1753:21)\n      at async D:\\work\\user_dashboard\\routes\\productRoutes.js:140:33 {\n    parent: Error: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''1'' at line 1\n        at Packet.asError (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n        at Query.execute (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n        at Connection.handlePacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:425:32)\n        at PacketParser.onPacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:75:12)\n        at PacketParser.executeStart (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n        at Socket.<anonymous> (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:82:25)\n        at Socket.emit (events.js:315:20)\n        at addChunk (internal/streams/readable.js:309:12)\n        at readableAddChunk (internal/streams/readable.js:284:9)\n        at Socket.Readable.push (internal/streams/readable.js:223:10) {\n      code: 'ER_PARSE_ERROR',\n      errno: 1064,\n      sqlState: '42000',\n      sqlMessage: \"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''1'' at line 1\",\n      sql: \"SELECT `id`, `name`, `description`, `price`, `image`, `status`, `quantity`, `category_id`, `subcategory_id`, `createdAt`, `updatedAt` FROM `Products` AS `Products` LIMIT '1';\",\n      parameters: undefined\n    },\n    original: Error: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''1'' at line 1\n        at Packet.asError (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n        at Query.execute (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n        at Connection.handlePacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:425:32)\n        at PacketParser.onPacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:75:12)\n        at PacketParser.executeStart (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n        at Socket.<anonymous> (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:82:25)\n        at Socket.emit (events.js:315:20)\n        at addChunk (internal/streams/readable.js:309:12)\n        at readableAddChunk (internal/streams/readable.js:284:9)\n        at Socket.Readable.push (internal/streams/readable.js:223:10) {\n      code: 'ER_PARSE_ERROR',\n      errno: 1064,\n      sqlState: '42000',\n      sqlMessage: \"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''1'' at line 1\",\n      sql: \"SELECT `id`, `name`, `description`, `price`, `image`, `status`, `quantity`, `category_id`, `subcategory_id`, `createdAt`, `updatedAt` FROM `Products` AS `Products` LIMIT '1';\",\n      parameters: undefined\n    },\n    sql: \"SELECT `id`, `name`, `description`, `price`, `image`, `status`, `quantity`, `category_id`, `subcategory_id`, `createdAt`, `updatedAt` FROM `Products` AS `Products` LIMIT '1';\",\n    parameters: undefined\n  }\n}","time":"2021-01-14T14:02:47.955Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":8828,"level":50,"msg":"Error accesssing getProducts {\n  error: DatabaseError [SequelizeDatabaseError]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''1'' at line 1\n      at Query.formatError (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:239:16)\n      at Query.run (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:54:18)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\sequelize.js:619:16\n      at async MySQLQueryInterface.select (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:938:12)\n      at async Function.findAll (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:1753:21)\n      at async D:\\work\\user_dashboard\\routes\\productRoutes.js:140:33 {\n    parent: Error: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''1'' at line 1\n        at Packet.asError (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n        at Query.execute (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n        at Connection.handlePacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:425:32)\n        at PacketParser.onPacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:75:12)\n        at PacketParser.executeStart (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n        at Socket.<anonymous> (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:82:25)\n        at Socket.emit (events.js:315:20)\n        at addChunk (internal/streams/readable.js:309:12)\n        at readableAddChunk (internal/streams/readable.js:284:9)\n        at Socket.Readable.push (internal/streams/readable.js:223:10) {\n      code: 'ER_PARSE_ERROR',\n      errno: 1064,\n      sqlState: '42000',\n      sqlMessage: \"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''1'' at line 1\",\n      sql: \"SELECT `id`, `name`, `description`, `price`, `image`, `status`, `quantity`, `category_id`, `subcategory_id`, `createdAt`, `updatedAt` FROM `Products` AS `Products` LIMIT '1';\",\n      parameters: undefined\n    },\n    original: Error: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''1'' at line 1\n        at Packet.asError (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n        at Query.execute (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n        at Connection.handlePacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:425:32)\n        at PacketParser.onPacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:75:12)\n        at PacketParser.executeStart (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n        at Socket.<anonymous> (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:82:25)\n        at Socket.emit (events.js:315:20)\n        at addChunk (internal/streams/readable.js:309:12)\n        at readableAddChunk (internal/streams/readable.js:284:9)\n        at Socket.Readable.push (internal/streams/readable.js:223:10) {\n      code: 'ER_PARSE_ERROR',\n      errno: 1064,\n      sqlState: '42000',\n      sqlMessage: \"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''1'' at line 1\",\n      sql: \"SELECT `id`, `name`, `description`, `price`, `image`, `status`, `quantity`, `category_id`, `subcategory_id`, `createdAt`, `updatedAt` FROM `Products` AS `Products` LIMIT '1';\",\n      parameters: undefined\n    },\n    sql: \"SELECT `id`, `name`, `description`, `price`, `image`, `status`, `quantity`, `category_id`, `subcategory_id`, `createdAt`, `updatedAt` FROM `Products` AS `Products` LIMIT '1';\",\n    parameters: undefined\n  }\n}","time":"2021-01-14T14:02:50.145Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":8828,"level":50,"msg":"Error accesssing getProducts {\n  error: DatabaseError [SequelizeDatabaseError]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''1'' at line 1\n      at Query.formatError (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:239:16)\n      at Query.run (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:54:18)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\sequelize.js:619:16\n      at async MySQLQueryInterface.select (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:938:12)\n      at async Function.findAll (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:1753:21)\n      at async D:\\work\\user_dashboard\\routes\\productRoutes.js:140:33 {\n    parent: Error: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''1'' at line 1\n        at Packet.asError (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n        at Query.execute (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n        at Connection.handlePacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:425:32)\n        at PacketParser.onPacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:75:12)\n        at PacketParser.executeStart (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n        at Socket.<anonymous> (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:82:25)\n        at Socket.emit (events.js:315:20)\n        at addChunk (internal/streams/readable.js:309:12)\n        at readableAddChunk (internal/streams/readable.js:284:9)\n        at Socket.Readable.push (internal/streams/readable.js:223:10) {\n      code: 'ER_PARSE_ERROR',\n      errno: 1064,\n      sqlState: '42000',\n      sqlMessage: \"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''1'' at line 1\",\n      sql: \"SELECT `id`, `name`, `description`, `price`, `image`, `status`, `quantity`, `category_id`, `subcategory_id`, `createdAt`, `updatedAt` FROM `Products` AS `Products` LIMIT '1';\",\n      parameters: undefined\n    },\n    original: Error: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''1'' at line 1\n        at Packet.asError (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n        at Query.execute (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n        at Connection.handlePacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:425:32)\n        at PacketParser.onPacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:75:12)\n        at PacketParser.executeStart (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n        at Socket.<anonymous> (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:82:25)\n        at Socket.emit (events.js:315:20)\n        at addChunk (internal/streams/readable.js:309:12)\n        at readableAddChunk (internal/streams/readable.js:284:9)\n        at Socket.Readable.push (internal/streams/readable.js:223:10) {\n      code: 'ER_PARSE_ERROR',\n      errno: 1064,\n      sqlState: '42000',\n      sqlMessage: \"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''1'' at line 1\",\n      sql: \"SELECT `id`, `name`, `description`, `price`, `image`, `status`, `quantity`, `category_id`, `subcategory_id`, `createdAt`, `updatedAt` FROM `Products` AS `Products` LIMIT '1';\",\n      parameters: undefined\n    },\n    sql: \"SELECT `id`, `name`, `description`, `price`, `image`, `status`, `quantity`, `category_id`, `subcategory_id`, `createdAt`, `updatedAt` FROM `Products` AS `Products` LIMIT '1';\",\n    parameters: undefined\n  }\n}","time":"2021-01-14T14:02:50.963Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":15852,"level":50,"msg":"Error accessing auth {\n  error: DatabaseError [SequelizeDatabaseError]: Unknown column 'user_id' in 'field list'\n      at Query.formatError (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:239:16)\n      at Query.run (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:54:18)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\sequelize.js:619:16\n      at async MySQLQueryInterface.select (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:938:12)\n      at async Function.findAll (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:1753:21)\n      at async Function.findOne (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:1917:12)\n      at async authenticateToken (D:\\work\\user_dashboard\\middleware\\auth.js:30:33) {\n    parent: Error: Unknown column 'user_id' in 'field list'\n        at Packet.asError (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n        at Query.execute (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n        at Connection.handlePacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:425:32)\n        at PacketParser.onPacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:75:12)\n        at PacketParser.executeStart (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n        at Socket.<anonymous> (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:82:25)\n        at Socket.emit (events.js:315:20)\n        at addChunk (internal/streams/readable.js:309:12)\n        at readableAddChunk (internal/streams/readable.js:284:9)\n        at Socket.Readable.push (internal/streams/readable.js:223:10) {\n      code: 'ER_BAD_FIELD_ERROR',\n      errno: 1054,\n      sqlState: '42S22',\n      sqlMessage: \"Unknown column 'user_id' in 'field list'\",\n      sql: 'SELECT `id`, `firstname`, `lastname`, `email`, `role_id`, `status`, `password`, `token`, `resetPasswordToken`, `createdAt`, `updatedAt`, `user_id` FROM `Users` AS `Users` WHERE `Users`.`id` = 1;',\n      parameters: undefined\n    },\n    original: Error: Unknown column 'user_id' in 'field list'\n        at Packet.asError (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n        at Query.execute (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n        at Connection.handlePacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:425:32)\n        at PacketParser.onPacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:75:12)\n        at PacketParser.executeStart (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n        at Socket.<anonymous> (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:82:25)\n        at Socket.emit (events.js:315:20)\n        at addChunk (internal/streams/readable.js:309:12)\n        at readableAddChunk (internal/streams/readable.js:284:9)\n        at Socket.Readable.push (internal/streams/readable.js:223:10) {\n      code: 'ER_BAD_FIELD_ERROR',\n      errno: 1054,\n      sqlState: '42S22',\n      sqlMessage: \"Unknown column 'user_id' in 'field list'\",\n      sql: 'SELECT `id`, `firstname`, `lastname`, `email`, `role_id`, `status`, `password`, `token`, `resetPasswordToken`, `createdAt`, `updatedAt`, `user_id` FROM `Users` AS `Users` WHERE `Users`.`id` = 1;',\n      parameters: undefined\n    },\n    sql: 'SELECT `id`, `firstname`, `lastname`, `email`, `role_id`, `status`, `password`, `token`, `resetPasswordToken`, `createdAt`, `updatedAt`, `user_id` FROM `Users` AS `Users` WHERE `Users`.`id` = 1;',\n    parameters: undefined\n  }\n}","time":"2021-01-15T06:07:45.923Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":12656,"level":50,"msg":"Error accessing loginUser {\n  error: DatabaseError [SequelizeDatabaseError]: Unknown column 'Users.user_id' in 'field list'\n      at Query.formatError (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:239:16)\n      at Query.run (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:54:18)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\sequelize.js:619:16\n      at async MySQLQueryInterface.select (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:938:12)\n      at async Function.findAll (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:1753:21)\n      at async Function.findOne (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:1917:12)\n      at async loginUser (D:\\work\\user_dashboard\\services\\userServices.js:163:31) {\n    parent: Error: Unknown column 'Users.user_id' in 'field list'\n        at Packet.asError (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n        at Query.execute (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n        at Connection.handlePacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:425:32)\n        at PacketParser.onPacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:75:12)\n        at PacketParser.executeStart (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n        at Socket.<anonymous> (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:82:25)\n        at Socket.emit (events.js:315:20)\n        at addChunk (internal/streams/readable.js:309:12)\n        at readableAddChunk (internal/streams/readable.js:284:9)\n        at Socket.Readable.push (internal/streams/readable.js:223:10) {\n      code: 'ER_BAD_FIELD_ERROR',\n      errno: 1054,\n      sqlState: '42S22',\n      sqlMessage: \"Unknown column 'Users.user_id' in 'field list'\",\n      sql: \"SELECT `Users`.`id`, `Users`.`firstname`, `Users`.`lastname`, `Users`.`email`, `Users`.`role_id`, `Users`.`status`, `Users`.`password`, `Users`.`token`, `Users`.`resetPasswordToken`, `Users`.`createdAt`, `Users`.`updatedAt`, `Users`.`user_id`, `Role`.`id` AS `Role.id`, `Role`.`name` AS `Role.name` FROM `Users` AS `Users` LEFT OUTER JOIN `Roles` AS `Role` ON `Users`.`role_id` = `Role`.`id` WHERE `Users`.`email` = 'nikku.a1998@gmail.com';\",\n      parameters: undefined\n    },\n    original: Error: Unknown column 'Users.user_id' in 'field list'\n        at Packet.asError (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n        at Query.execute (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n        at Connection.handlePacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:425:32)\n        at PacketParser.onPacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:75:12)\n        at PacketParser.executeStart (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n        at Socket.<anonymous> (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:82:25)\n        at Socket.emit (events.js:315:20)\n        at addChunk (internal/streams/readable.js:309:12)\n        at readableAddChunk (internal/streams/readable.js:284:9)\n        at Socket.Readable.push (internal/streams/readable.js:223:10) {\n      code: 'ER_BAD_FIELD_ERROR',\n      errno: 1054,\n      sqlState: '42S22',\n      sqlMessage: \"Unknown column 'Users.user_id' in 'field list'\",\n      sql: \"SELECT `Users`.`id`, `Users`.`firstname`, `Users`.`lastname`, `Users`.`email`, `Users`.`role_id`, `Users`.`status`, `Users`.`password`, `Users`.`token`, `Users`.`resetPasswordToken`, `Users`.`createdAt`, `Users`.`updatedAt`, `Users`.`user_id`, `Role`.`id` AS `Role.id`, `Role`.`name` AS `Role.name` FROM `Users` AS `Users` LEFT OUTER JOIN `Roles` AS `Role` ON `Users`.`role_id` = `Role`.`id` WHERE `Users`.`email` = 'nikku.a1998@gmail.com';\",\n      parameters: undefined\n    },\n    sql: \"SELECT `Users`.`id`, `Users`.`firstname`, `Users`.`lastname`, `Users`.`email`, `Users`.`role_id`, `Users`.`status`, `Users`.`password`, `Users`.`token`, `Users`.`resetPasswordToken`, `Users`.`createdAt`, `Users`.`updatedAt`, `Users`.`user_id`, `Role`.`id` AS `Role.id`, `Role`.`name` AS `Role.name` FROM `Users` AS `Users` LEFT OUTER JOIN `Roles` AS `Role` ON `Users`.`role_id` = `Role`.`id` WHERE `Users`.`email` = 'nikku.a1998@gmail.com';\",\n    parameters: undefined\n  }\n}","time":"2021-01-15T06:28:15.030Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":13768,"level":50,"msg":"Error accesssing editSubcategory { error: \"Cannot read property 'dataValues' of undefined\" }","time":"2021-01-15T06:53:44.478Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":6548,"level":50,"msg":"Error accesssing editProduct {\n  error: TypeError: Cannot read property 'dataValues' of undefined\n      at D:\\work\\user_dashboard\\routes\\productRoutes.js:64:115\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n}","time":"2021-01-15T07:03:09.897Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":18416,"level":50,"msg":"Error accesssing editProduct { error: 'id doesnt exist' }","time":"2021-01-15T07:12:26.902Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":7556,"level":50,"msg":"Error accesssing getProducts {\n  error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n      at ServerResponse.setHeader (_http_outgoing.js:558:11)\n      at ServerResponse.header (D:\\work\\user_dashboard\\node_modules\\express\\lib\\response.js:771:10)\n      at ServerResponse.send (D:\\work\\user_dashboard\\node_modules\\express\\lib\\response.js:170:12)\n      at ServerResponse.json (D:\\work\\user_dashboard\\node_modules\\express\\lib\\response.js:267:15)\n      at ServerResponse.send (D:\\work\\user_dashboard\\node_modules\\express\\lib\\response.js:158:21)\n      at D:\\work\\user_dashboard\\routes\\productRoutes.js:149:29\n      at processTicksAndRejections (internal/process/task_queues.js:93:5) {\n    code: 'ERR_HTTP_HEADERS_SENT'\n  }\n}","time":"2021-01-15T07:17:42.600Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":7556,"level":50,"msg":"Error accesssing createProduct {\n  error: UniqueConstraintError [SequelizeUniqueConstraintError]: Validation error\n      at Query.formatError (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:218:16)\n      at Query.run (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:54:18)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\sequelize.js:619:16\n      at async MySQLQueryInterface.insert (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:734:21)\n      at async model.save (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:3954:35)\n      at async Function.create (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:2207:12)\n      at async D:\\work\\user_dashboard\\routes\\productRoutes.js:23:32 {\n    errors: [ [ValidationErrorItem] ],\n    fields: { 'products.name': 'puma casual shoes' },\n    parent: Error: Duplicate entry 'puma casual shoes' for key 'products.name'\n        at Packet.asError (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n        at Execute.execute (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n        at Connection.handlePacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:425:32)\n        at PacketParser.onPacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:75:12)\n        at PacketParser.executeStart (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n        at Socket.<anonymous> (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:82:25)\n        at Socket.emit (events.js:315:20)\n        at addChunk (internal/streams/readable.js:309:12)\n        at readableAddChunk (internal/streams/readable.js:284:9)\n        at Socket.Readable.push (internal/streams/readable.js:223:10) {\n      code: 'ER_DUP_ENTRY',\n      errno: 1062,\n      sqlState: '23000',\n      sqlMessage: \"Duplicate entry 'puma casual shoes' for key 'products.name'\",\n      sql: 'INSERT INTO `Products` (`id`,`name`,`description`,`price`,`status`,`quantity`,`category_id`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?,?);',\n      parameters: [Array]\n    },\n    original: Error: Duplicate entry 'puma casual shoes' for key 'products.name'\n        at Packet.asError (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n        at Execute.execute (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n        at Connection.handlePacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:425:32)\n        at PacketParser.onPacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:75:12)\n        at PacketParser.executeStart (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n        at Socket.<anonymous> (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:82:25)\n        at Socket.emit (events.js:315:20)\n        at addChunk (internal/streams/readable.js:309:12)\n        at readableAddChunk (internal/streams/readable.js:284:9)\n        at Socket.Readable.push (internal/streams/readable.js:223:10) {\n      code: 'ER_DUP_ENTRY',\n      errno: 1062,\n      sqlState: '23000',\n      sqlMessage: \"Duplicate entry 'puma casual shoes' for key 'products.name'\",\n      sql: 'INSERT INTO `Products` (`id`,`name`,`description`,`price`,`status`,`quantity`,`category_id`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?,?);',\n      parameters: [Array]\n    },\n    sql: 'INSERT INTO `Products` (`id`,`name`,`description`,`price`,`status`,`quantity`,`category_id`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?,?);'\n  }\n}","time":"2021-01-15T07:19:59.003Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":7556,"level":50,"msg":"Error accesssing subcategory { error: 'Validation error' }","time":"2021-01-15T07:20:07.174Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":7556,"level":50,"msg":"Error accesssing editProduct { error: 'id doesnt exist' }","time":"2021-01-15T07:21:07.323Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":3216,"level":50,"msg":"Error accesssing createProduct {\n  error: UniqueConstraintError [SequelizeUniqueConstraintError]: Validation error\n      at Query.formatError (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:218:16)\n      at Query.run (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:54:18)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\sequelize.js:619:16\n      at async MySQLQueryInterface.insert (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:734:21)\n      at async model.save (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:3954:35)\n      at async Function.create (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:2207:12)\n      at async D:\\work\\user_dashboard\\routes\\productRoutes.js:23:32 {\n    errors: [ [ValidationErrorItem] ],\n    fields: { 'products.name': 'puma casual shoes' },\n    parent: Error: Duplicate entry 'puma casual shoes' for key 'products.name'\n        at Packet.asError (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n        at Execute.execute (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n        at Connection.handlePacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:425:32)\n        at PacketParser.onPacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:75:12)\n        at PacketParser.executeStart (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n        at Socket.<anonymous> (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:82:25)\n        at Socket.emit (events.js:315:20)\n        at addChunk (internal/streams/readable.js:309:12)\n        at readableAddChunk (internal/streams/readable.js:284:9)\n        at Socket.Readable.push (internal/streams/readable.js:223:10) {\n      code: 'ER_DUP_ENTRY',\n      errno: 1062,\n      sqlState: '23000',\n      sqlMessage: \"Duplicate entry 'puma casual shoes' for key 'products.name'\",\n      sql: 'INSERT INTO `Products` (`id`,`name`,`description`,`price`,`status`,`quantity`,`category_id`,`subcategory_id`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?,?,?);',\n      parameters: [Array]\n    },\n    original: Error: Duplicate entry 'puma casual shoes' for key 'products.name'\n        at Packet.asError (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packets\\packet.js:712:17)\n        at Execute.execute (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\commands\\command.js:28:26)\n        at Connection.handlePacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:425:32)\n        at PacketParser.onPacket (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:75:12)\n        at PacketParser.executeStart (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\packet_parser.js:75:16)\n        at Socket.<anonymous> (D:\\work\\user_dashboard\\node_modules\\mysql2\\lib\\connection.js:82:25)\n        at Socket.emit (events.js:315:20)\n        at addChunk (internal/streams/readable.js:309:12)\n        at readableAddChunk (internal/streams/readable.js:284:9)\n        at Socket.Readable.push (internal/streams/readable.js:223:10) {\n      code: 'ER_DUP_ENTRY',\n      errno: 1062,\n      sqlState: '23000',\n      sqlMessage: \"Duplicate entry 'puma casual shoes' for key 'products.name'\",\n      sql: 'INSERT INTO `Products` (`id`,`name`,`description`,`price`,`status`,`quantity`,`category_id`,`subcategory_id`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?,?,?);',\n      parameters: [Array]\n    },\n    sql: 'INSERT INTO `Products` (`id`,`name`,`description`,`price`,`status`,`quantity`,`category_id`,`subcategory_id`,`createdAt`,`updatedAt`) VALUES (DEFAULT,?,?,?,?,?,?,?,?,?);'\n  }\n}","time":"2021-01-15T07:22:47.110Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":16960,"level":50,"msg":"Error response from authorized { error: 'unauthorized role' }","time":"2021-01-15T07:29:08.317Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":16960,"level":50,"msg":"Error accessing createUser {\n  error: ValidationError [SequelizeValidationError]: notNull Violation: Users.role_id cannot be null\n      at InstanceValidator._validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:78:13)\n      at processTicksAndRejections (internal/process/task_queues.js:93:5)\n      at async InstanceValidator._validateAndRunHooks (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:111:7)\n      at async InstanceValidator.validate (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\instance-validator.js:93:12)\n      at async model.save (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:3877:7)\n      at async Function.create (D:\\work\\user_dashboard\\node_modules\\sequelize\\lib\\model.js:2207:12)\n      at async createUser (D:\\work\\user_dashboard\\services\\userServices.js:64:28) {\n    errors: [ [ValidationErrorItem] ]\n  }\n}","time":"2021-01-15T07:29:44.049Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":16960,"level":50,"msg":"Error accessing auth {\n  error: TypeError: Cannot read property 'split' of undefined\n      at authenticateToken (D:\\work\\user_dashboard\\middleware\\auth.js:26:54)\n      at Layer.handle [as handle_request] (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\layer.js:95:5)\n      at next (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\route.js:137:13)\n      at Route.dispatch (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\route.js:112:3)\n      at Layer.handle [as handle_request] (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\layer.js:95:5)\n      at D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:281:22\n      at Function.process_params (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:335:12)\n      at next (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:275:10)\n      at Function.handle (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:174:3)\n      at router (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:47:12)\n}","time":"2021-01-15T07:30:26.605Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":16960,"level":50,"msg":"Error accessing auth {\n  error: TypeError: Cannot read property 'split' of undefined\n      at authenticateToken (D:\\work\\user_dashboard\\middleware\\auth.js:26:54)\n      at Layer.handle [as handle_request] (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\layer.js:95:5)\n      at next (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\route.js:137:13)\n      at Route.dispatch (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\route.js:112:3)\n      at Layer.handle [as handle_request] (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\layer.js:95:5)\n      at D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:281:22\n      at Function.process_params (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:335:12)\n      at next (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:275:10)\n      at Function.handle (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:174:3)\n      at router (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:47:12)\n}","time":"2021-01-15T07:30:52.523Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":16960,"level":50,"msg":"Error accessing auth {\n  error: TypeError: Cannot read property 'split' of undefined\n      at authenticateToken (D:\\work\\user_dashboard\\middleware\\auth.js:26:54)\n      at Layer.handle [as handle_request] (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\layer.js:95:5)\n      at next (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\route.js:137:13)\n      at Route.dispatch (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\route.js:112:3)\n      at Layer.handle [as handle_request] (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\layer.js:95:5)\n      at D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:281:22\n      at Function.process_params (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:335:12)\n      at next (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:275:10)\n      at Function.handle (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:174:3)\n      at router (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:47:12)\n}","time":"2021-01-15T07:30:55.689Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":16960,"level":50,"msg":"Error accessing auth {\n  error: TypeError: Cannot read property 'split' of undefined\n      at authenticateToken (D:\\work\\user_dashboard\\middleware\\auth.js:26:54)\n      at Layer.handle [as handle_request] (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\layer.js:95:5)\n      at next (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\route.js:137:13)\n      at Route.dispatch (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\route.js:112:3)\n      at Layer.handle [as handle_request] (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\layer.js:95:5)\n      at D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:281:22\n      at Function.process_params (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:335:12)\n      at next (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:275:10)\n      at Function.handle (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:174:3)\n      at router (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:47:12)\n}","time":"2021-01-15T07:31:10.435Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":16960,"level":50,"msg":"Error accessing auth {\n  error: TypeError: Cannot read property 'split' of undefined\n      at authenticateToken (D:\\work\\user_dashboard\\middleware\\auth.js:26:54)\n      at Layer.handle [as handle_request] (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\layer.js:95:5)\n      at next (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\route.js:137:13)\n      at Route.dispatch (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\route.js:112:3)\n      at Layer.handle [as handle_request] (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\layer.js:95:5)\n      at D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:281:22\n      at Function.process_params (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:335:12)\n      at next (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:275:10)\n      at Function.handle (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:174:3)\n      at router (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:47:12)\n}","time":"2021-01-15T07:31:30.275Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":512,"level":50,"msg":"Error accessing auth {\n  error: TypeError: Cannot read property 'split' of undefined\n      at authenticateToken (D:\\work\\user_dashboard\\middleware\\auth.js:26:54)\n      at Layer.handle [as handle_request] (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\layer.js:95:5)\n      at next (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\route.js:137:13)\n      at Route.dispatch (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\route.js:112:3)\n      at Layer.handle [as handle_request] (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\layer.js:95:5)\n      at D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:281:22\n      at param (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:354:14)\n      at param (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:365:14)\n      at Function.process_params (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:410:3)\n      at next (D:\\work\\user_dashboard\\node_modules\\express\\lib\\router\\index.js:275:10)\n}","time":"2021-01-15T07:32:08.387Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":16232,"level":50,"msg":"Error response from addtocart { error: undefined }","time":"2021-01-15T07:38:49.147Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":13744,"level":50,"msg":"Error accesssing removefromcart { error: 'ProductsCarts is not associated to Carts!' }","time":"2021-01-15T08:02:47.093Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":9632,"level":50,"msg":"Error accesssing removefromcart { error: 'ProductsCarts is not associated to Carts!' }","time":"2021-01-15T08:04:54.931Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":15044,"level":50,"msg":"Error accesssing removefromcart { error: 'Carts is not associated to ProductsCarts!' }","time":"2021-01-15T08:08:15.595Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":15044,"level":50,"msg":"Error accesssing removefromcart { error: 'Carts is not associated to ProductsCarts!' }","time":"2021-01-15T08:08:17.757Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":8696,"level":50,"msg":"Error accesssing removefromcart { error: 'Carts is not associated to ProductsCarts!' }","time":"2021-01-15T08:08:50.728Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":8696,"level":50,"msg":"Error accesssing removefromcart { error: 'Carts is not associated to ProductsCarts!' }","time":"2021-01-15T08:08:51.938Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":10864,"level":50,"msg":"Error accesssing removefromcart { error: 'Carts is not associated to ProductsCarts!' }","time":"2021-01-15T08:15:17.939Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":10864,"level":50,"msg":"Error accesssing removefromcart { error: 'Carts is not associated to ProductsCarts!' }","time":"2021-01-15T08:15:19.127Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":13928,"level":50,"msg":"Error accesssing removefromcart { error: \"Unknown column 'ProductsCarts.ProductId' in 'field list'\" }","time":"2021-01-15T08:25:59.447Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":12264,"level":50,"msg":"Error accesssing add to cart { error: 'addCart is not defined' }","time":"2021-01-15T08:46:28.069Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":14532,"level":50,"msg":"Error accesssing add to cart { error: 'addCart is not defined' }","time":"2021-01-15T08:47:46.281Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":11164,"level":50,"msg":"Error accesssing add to cart { error: \"Cannot read property 'dataValues' of null\" }","time":"2021-01-15T08:54:03.833Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":11164,"level":50,"msg":"Error accesssing add to cart { error: \"Cannot read property 'dataValues' of null\" }","time":"2021-01-15T08:54:22.531Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":7832,"level":50,"msg":"Error accesssing add to cart { error: 'addCart is not defined' }","time":"2021-01-15T08:54:47.305Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":16868,"level":50,"msg":"Error accesssing add to cart { error: 'addCart is not defined' }","time":"2021-01-15T08:55:20.785Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":18080,"level":50,"msg":"Error accesssing add to cart { error: 'addCart is not defined' }","time":"2021-01-15T08:56:36.145Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":14828,"level":50,"msg":"Error accesssing add to cart { error: 'addCart is not defined' }","time":"2021-01-15T08:58:16.012Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":17444,"level":50,"msg":"Error accesssing add to cart { error: 'addCart is not defined' }","time":"2021-01-15T08:59:00.000Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":3392,"level":50,"msg":"Error accesssing add to cart { error: 'addCart is not defined' }","time":"2021-01-15T09:00:21.490Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":2152,"level":50,"msg":"Error accesssing add to cart { error: 'mapToProduct is not defined' }","time":"2021-01-15T09:01:41.328Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":6772,"level":50,"msg":"Error accesssing add to cart { error: 'Validation error' }","time":"2021-01-15T09:02:20.850Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":10356,"level":50,"msg":"Error accesssing add to cart { error: 'Validation error' }","time":"2021-01-15T09:04:01.011Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":10356,"level":50,"msg":"Error accesssing add to cart { error: 'Validation error' }","time":"2021-01-15T09:04:02.405Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":17784,"level":50,"msg":"Error accesssing add to cart { error: \"Unknown column 'product_id' in 'field list'\" }","time":"2021-01-15T09:07:18.697Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":4972,"level":50,"msg":"Error accesssing add to cart { error: 'Validation error' }","time":"2021-01-15T09:09:13.422Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":14872,"level":50,"msg":"Error accesssing removefromcart { error: \"Unknown column 'ProductsCarts->Cart.' in 'field list'\" }","time":"2021-01-15T09:22:13.691Z","v":0}
{"name":"userLogger","hostname":"LAPTOP-QV5SU20B","pid":15432,"level":50,"msg":"Error accesssing removefromcart { error: \"Unknown column 'ProductsCarts->Cart.' in 'field list'\" }","time":"2021-01-15T09:27:45.264Z","v":0}
